en/docs/install-and-setup/install/add-centralized-observability-in-icp.md

```
@@ -0,0 +1,535 @@
+# Adding Centralized Observability in the Integration Control Plane
+
+ICP provides centralized observability for MI runtimes. Application logs and per-request analytics are collected via Fluent Bit, stored in OpenSearch, and displayed in the ICP Console.
+
+## Architecture
+
+<a href="{{base_path}}/assets/img/integrate/observability/observability-icp.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/observability/observability-icp.png" width="1000"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)